enterprise-beans/4.0/_index.md

```diff
@@ -33,20 +33,20 @@ Jakarta Enterprise Beans defines an architecture for the development and deploym
 
 [Jakarta Enterprise Bean 4.0 Release Record and Plan]()
 
-The Specification Committee Ballot concluded successfully on 2020-mm-dd with the following results.
+The Specification Committee Ballot concluded successfully on 2020-03-04 with the following results.
 
 | Representative                                 | Representative for: | Vote |
 |------------------------------------------------|---------------------|------|
-| Kenji Kazumura, Michael DeNicola               | Fujitsu             |      |
-| Dan Bandera, Kevin Sutter                      | IBM                 |      |
-| Bill Shannon, Ed Bratt                         | Oracle              |      |
-| Mark Wareham, Steve Millidge                   | Payara              |      |
-| Scott Stark, Mark Little                       | Red Hat             |      |
-| David Blevins, Cesar Hernandez                 | Tomitribe           |      |
-| Ivar Grimstad                                  | EE4J PMC            |      |
+| Kenji Kazumura, Michael DeNicola               | Fujitsu             |   1  |
+| Dan Bandera, Kevin Sutter                      | IBM                 |   1  |
+| Bill Shannon, Ed Bratt                         | Oracle              |   1  |
+| Mark Wareham, Steve Millidge                   | Payara              |   1  |
+| Scott Stark, Mark Little                       | Red Hat             |   1  |
+| David Blevins, Cesar Hernandez                 | Tomitribe           |   1  |
+| Ivar Grimstad                                  | EE4J PMC            |   1  |
 | Alex Theedom                                   | Participant Members |      |
-| Werner Keil                                    | Committer Members   |      |
-|                                                | Total               |      |
+| Werner Keil                                    | Committer Members   |   1  |
+|                                                | Total               |   8  |
 
 The ballot was run in the [jakarta.ee-spec mailing list]()
```

Comment:
@waynebeaton could we also include a link to the final ballot message in the public spec mailing list?